client/client.py

```
@@ -218,36 +252,35 @@ def on_ice_state():
         channel = pc.createDataChannel("chat")
         self.channels[host_id] = channel
-        print('Got here #3')
+        key = Fernet.generate_key()
+        self.keys[host_id] = key
 
         @channel.on("open")
         def on_open():
             logging.info("DataChannel open to host — start typing messages")
-            asyncio.create_task(self.async_input_loop(channel))
-            channel.send(f"[{self.name}] Joined the room")
+            asyncio.create_task(self.async_input_loop(host_id))
```

Ruff (0.13.1)

Store a reference to the return value of `asyncio.create_task`

(RUF006)